### PR TITLE
feat: evaluate environment variables in the args of gradle

### PR DESCRIPTION
**Pull request by vLia**
## Pull request description 

https://github.com/kubeshop/testkube/issues/4456

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: contrib/executor/gradle/pkg/runner/runner.go [hunk 1]
```diff
@@ -131,9 +131,9 @@ func (r *GradleRunner) Run(ctx context.Context, execution testkube.Execution) (r
 		if args[i] == "<projectDir>" {
 			args[i] = project
 		}
+		args[i] = os.ExpandEnv(args[i])
 	}
 
-	output.PrintEvent("Running task: "+task, directory, gradleCommand, args)
```
Comment (vsukhin) [inline contrib/executor/gradle/pkg/runner/runner.go:136]: why do we remove this - duplication?

**Review by vsukhin — APPROVED** [review]
one question

File: contrib/executor/gradle/pkg/runner/runner.go (hunk 1 above)
Comment (vLia) [reply contrib/executor/gradle/pkg/runner/runner.go:136]: Both duplication and it prints out the secret variables. The command with arguments will be printed out in the following function with obfuscated secrets, and I also checked that there are no errors before it.

**Comment by vsukhin** [comment]
ok, also worth to create a ticket with similar improvements for all our prebuilt executors

**Pull request merged**